week3/homework/src/package.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "src",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

since you don't have an `index.js` this will be a useful tip for the future, we can write our own command to run it easier:
```suggestion
    "start": "node app.js"
```
now with one command you can run your application as `npm start`
the left side is the script name, the right side is what ever you run in your terminal which is `node app.js`